### PR TITLE
PR: IntuneAntivirusPolicyWindows10SettingCatalog Fixes 3818 and 3955

### PR DESCRIPTION
**Pull request by William-Francillette**
#### This Pull Request (PR) fixes the following issues

* IntuneAntivirusPolicyWindows10SettingCatalog
  * Skipped settingValueTemplateReference and settingInstanceTemplateReference for severethreats, highseveritythreats, moderateseveritythreats, lowseveritythreats as per API requirements observed in the Intune portal
  FIXES [#3818](https://github.com/microsoft/Microsoft365DSC/issues/3818)

### CODE REVIEW

File: Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAntivirusPolicyWindows10SettingCatalog/MSFT_IntuneAntivirusPolicyWindows10SettingCatalog.psm1 [hunk 1]
```diff
@@ -1590,7 +1590,7 @@ function New-IntuneDeviceConfigurationPolicy
             'settings'          = $Settings
         }
         $body = $policy | ConvertTo-Json -Depth 20
-        Write-Verbose -Message $body
```
Comment (ykuijs) [inline Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAntivirusPolicyWindows10SettingCatalog/MSFT_IntuneAntivirusPolicyWindows10SettingCatalog.psm1:1593]: I think this is here for troubleshooting purposes and should be commented out, correct?

**Review by ykuijs — COMMENTED** [review]
One small comment

File: Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAntivirusPolicyWindows10SettingCatalog/MSFT_IntuneAntivirusPolicyWindows10SettingCatalog.psm1 (hunk 1 above)
Comment (William-Francillette) [reply Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAntivirusPolicyWindows10SettingCatalog/MSFT_IntuneAntivirusPolicyWindows10SettingCatalog.psm1:1593]: yep already commented

**Comment by William-Francillette** [comment]
@ykuijs all sorted

**Comment by William-Francillette** [comment]
> @William-Francillette would this PR also address #3955 ?

@andikrueger just pushed the fix

**Pull request merged**